ast/misc.go

```diff
@@ -721,6 +721,40 @@ func (n *SetPwdStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+type ChangeStmt struct {
+	stmtNode
+
+	NodeType  string
```

Comment:
Should we use an `enum` for this?

Reply:
i already changed, but i need you confirm it.